react/__tests__/components/Greeting.test.js

```diff
@@ -27,7 +28,7 @@ describe('<Greeting /> component', () => {
     }
 
     const comp = (
-      <MockedProvider cache={cache} resolvers={{}}>
+      <MockedProvider cache={cache}>
```

Comment:
Render from `@vtex/test-tools/react` already has a MockedProvider.
See in: https://github.com/vtex/test-tools